### PR TITLE
Some small changes

### PR DESCRIPTION
**Pull request by nightism**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -188,72 +183,73 @@ TaskTracker makes time-management smart and simple for you!
...
 
 	 * `exit`
 
         Close the Task-tracker.
 
-16. Refer to the [Features](#features) section below for details of each
-    command.
+15. For more details of each 
+     command.
+please refer to the [Features](#features) section below.
```
Comment (pyokagan) [inline docs/UserGuide.md:212]: Eh... will this format properly... fingers crossed.

**Review by pyokagan — APPROVED** [review]
OK

**Comment by pyokagan** [comment]
@nightism Ugh I just noticed that you did not apply your `Format:` thing to all commands. 

Please follow up on this. Either remove all the `Format:` things or do it for all.

**Comment by nightism** [comment]
@pyokagan please check if this branch can be merged so that Sherry can do further revision

**Comment by pyokagan** [comment]
Merged in 279715e24dcb8714cd4cf5355d1afde61087ac17

**Pull request closed without merging**